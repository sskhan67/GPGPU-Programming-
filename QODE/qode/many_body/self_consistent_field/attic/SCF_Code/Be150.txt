Pre-SCF process for electrons
INPUT FILE NAME: be2.in
Charge: 0 	 Multiplicity: 1
Exchange: HF 
Basis: sto-test 
Correlation: mp2
[0.0, 0.0, 0.0, 30.167871, 0.15432897]
[0.0, 0.0, 0.0, 5.4951153, 0.53532814]
[0.0, 0.0, 0.0, 1.4871927, 0.44463454]
[0.0, 0.0, 0.0, 1.3148331, -0.09996723]
[0.0, 0.0, 0.0, 0.3055389, 0.39951283]
[0.0, 0.0, 0.0, 0.0993707, 0.70011547]
[0.0, 0.0, 3.69, 30.167871, 0.15432897]
[0.0, 0.0, 3.69, 5.4951153, 0.53532814]
[0.0, 0.0, 3.69, 1.4871927, 0.44463454]
[0.0, 0.0, 3.69, 1.3148331, -0.09996723]
[0.0, 0.0, 3.69, 0.3055389, 0.39951283]
[0.0, 0.0, 3.69, 0.0993707, 0.70011547]
# of gaussians in each orbital:
[3, 3, 3, 3]
Orbital Types:
['S', 'S', 'S', 'S']
Input Symbols:
['Be', 'Be']
Number of electrons:
8
nuclei list and charge:
[[0.0, 0.0, 0.0], [0.0, 0.0, 3.69]]
[4, 4]
a e-: 4 b e-: 4
guess:
5.113988e-11   -7.480950e-01   6.635916e-01   -4.372632e-13   0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   
8.144754e-01   4.665821e-11   3.699994e-13   5.801980e-01   0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   
5.801980e-01   3.323373e-11   -5.237067e-13   -8.144754e-01   0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   
-4.536005e-11   6.635916e-01   7.480950e-01   -4.975470e-13   0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   
0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   5.113988e-11   -7.480950e-01   6.635916e-01   -4.372632e-13   
0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   8.144754e-01   4.665821e-11   3.699994e-13   5.801980e-01   
0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   5.801980e-01   3.323373e-11   -5.237067e-13   -8.144754e-01   
0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   -4.536005e-11   6.635916e-01   7.480950e-01   -4.975470e-13   
occa [0, 1, 2, 3] occb [4, 5, 6, 7]
fock.occ [0, 1, 2, 3, 4, 5, 6, 7]
Cycle 1
conv: vec = 1 energy = 8
Cycle 2
conv: vec = 2 energy = 8
Cycle 3
conv: vec = 3 energy = 5
Cycle 4
conv: vec = 4 energy = 4
Cycle 5
conv: vec = 5 energy = 8
Cycle 6
conv: vec = 6 energy = 8
Cycle 7
conv: vec = 7 energy = 1
Cycle 8
conv: vec = 8 energy = 0
Cycle 9
conv: vec = 8 energy = 8
U
4.285558e-11   7.480950e-01   6.635916e-01   4.373619e-13   0.000000e+00   0.000000e+00   -0.000000e+00   0.000000e+00   
8.144754e-01   -4.665823e-11   3.823608e-13   -5.801980e-01   0.000000e+00   0.000000e+00   -0.000000e+00   0.000000e+00   
5.801980e-01   -3.323367e-11   -5.409934e-13   8.144754e-01   0.000000e+00   0.000000e+00   -0.000000e+00   0.000000e+00   
-3.801141e-11   -6.635916e-01   7.480950e-01   4.975855e-13   0.000000e+00   0.000000e+00   -0.000000e+00   0.000000e+00   
0.000000e+00   0.000000e+00   -0.000000e+00   0.000000e+00   -4.285566e-11   7.480950e-01   6.635916e-01   4.373619e-13   
0.000000e+00   0.000000e+00   -0.000000e+00   0.000000e+00   -8.144754e-01   -4.665823e-11   3.823608e-13   -5.801980e-01   
0.000000e+00   0.000000e+00   -0.000000e+00   0.000000e+00   -5.801980e-01   -3.323367e-11   -5.409934e-13   8.144754e-01   
0.000000e+00   0.000000e+00   -0.000000e+00   0.000000e+00   3.801142e-11   -6.635916e-01   7.480950e-01   4.975855e-13   
HF = -28.7014182481
KE = 29.6986501665
NAE = -72.7422977361
ERE = 12.0476939896
