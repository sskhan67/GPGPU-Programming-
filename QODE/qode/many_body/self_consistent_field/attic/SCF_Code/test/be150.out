/home/group/software/Q-Chem/qchem-20jun2013/exe/qcprog.exe .be2.in.12333.qcin.1 /scratch/yhliu/Q-Chem/qchem12333/

                  Welcome to Q-Chem
     A Quantum Leap Into The Future Of Chemistry

 Y. Shao, L. Fusti-Molnar, Y. Jung, J. Kussmann, C. Ochsenfeld,
 S. T. Brown, A. T. B. Gilbert, L. V. Slipchenko,S. V. Levchenko,
 D. P. O'Neill, R. A. DiStasio Jr., R. C. Lochan, T. Wang,
 G. J. O. Beran, N. A. Besley, J. M. Herbert, C. Y. Lin,
 T. Van Voorhis, S. H. Chien, A. Sodt, R. P. Steele, 
 V. A. Rassolov, P. E. Maslen, P. P. Korambath, R. D. Adamson,
 B. Austin, J. Baker, E. F. C. Byrd, H. Dachsel, R. J. Doerksen,
 A. Dreuw, B. D. Dunietz, A. D. Dutoi, T. R. Furlani,
 S. R. Gwaltney, A. Heyden, S. Hirata, C.-P. Hsu, G. Kedziora,
 R. Z. Khaliullin, P. Klunzinger, A. M. Lee, M. S. Lee, W. Liang,
 I. Lotan, N. Nair, B. Peters, E. I. Proynov, P. A. Pieniazek, 
 Y. M. Rhee, J. Ritchie, E. Rosta, C. D. Sherrill, 
 A. C. Simmonett, J. E. Subotnik, H. L. Woodcock III,
 W. Zhang, A. T. Bell, A. K. Chakraborty, D. M. Chipman,
 F. J. Keil, A. Warshel, W. J. Hehre, H. F. Schaefer III,
 J. Kong, A. I. Krylov, P. M. W. Gill, M. Head-Gordon,

 Additional authors for Version 3.1: 
 Z. Gan, Y. Zhao, N. E. Schultz, D. Truhlar, E. Epifanovsky and M. Oana.

 Additional authors for Version 3.2: 
 R. Baer, B. R. Brooks, D. Casanova, J.-D. Chai, C.-L. Cheng, 
 C. Cramer, D. Crittenden, A. Ghysels, G. Hawkins, E. G. Hohenstein, 
 C. Kelley, W. Kurlancheek, D. Liotard, E. Livshits, P. Manohar, 
 A. Marenich, D. Neuhauser, R. Olson, M. A. Rohrdanz, K. S. Thanthiriwatte, 
 A. J. W. Thom, V. Vanovschi, C. F. Williams, Q. Wu and Z.-Q. You. 

 Additional authors for Version 4.0: 
 A. Aspuru-Guzik, C. Chang, R. G. Edgar, E. Sundstrom, J. Parkhill, K. Lawler,
 M. Gordon, M. Schmidt, N. Shenvi, D. Lambrecht, M. Goldey, R. Olivares-Amaya,
 Y. Bernard, L. Vogt, M. Watson, J. Liu, S. Yeganeh, B. Kaduk, O. Vydrov, X. Xu, 
 I. Kaliman, K. Khistyaev, N. Russ, I.Y. Zhang, W.A. Goddard III, F. Liu, R. King ,
 A. Landau, M. Wormit, A. Dreuw, M. Diedenhofen, A. Klamt, A.W. Lange, D. Ghosh,
 D. Kosenkov, T. Kus, A. Landau, D. Zuev, J. Deng, S.P. Mao, Y.C. Su, D. Small,
 L. D. Jacobson
 
 Q-Chem, Version 4.0.1, Q-Chem, Inc., Pittsburgh, PA (2007).


 Intel X86 EM64T Linux Version 4.1.0.1 

 Q-chem begins on Mon Jun 16 16:40:34 2014  

medusa
Host: 0

     Scratch files written to /scratch/yhliu/Q-Chem/qchem12333//
 Jun13 |home|group|software|Q-Chem|qchem-20jun2013|libmdc|qstr_info.C -1

theFileMan(): MAXOPENFILES=974 MAX_SUB_FILE_NUM=16
Maximum size of a physical file is 2.0 GB, maximum size of a tmp-file is 32.0 GB

Finally everything over in PARseQInput

--------------------------------------------------------------
User input:
--------------------------------------------------------------
$molecule
0 1 
Be   0.0   0.0   0.0
Be   0.0   0.0   3.69
$end

$rem
exchange      HF
basis         gen
$end

$basis
BE    0
S   3  1.00
     30.16787100      0.15432897
      5.49511530      0.53532814
      1.48719270      0.44463454
S   3  1.00
      1.31483310     -0.09996723      
      0.30553890      0.39951283      
      0.09937070      0.70011547                                            
****
$end

--------------------------------------------------------------
 ----------------------------------------------------
       Standard Nuclear Orientation (Angstroms)
    I     Atom         X            Y            Z
 ----------------------------------------------------
    1      Be      0.000000     0.000000    -1.845000
    2      Be      0.000000     0.000000     1.845000
 ----------------------------------------------------
 Molecular Point Group                 D*h   NOp =***
 Largest Abelian Subgroup              D2h   NOp =  1
 Nuclear Repulsion Energy =     2.2945353314 hartrees
 There are        4 alpha and        4 beta electrons

 Q-Chem warning in module /home/group/software/Q-Chem/qchem-20jun2013/forms1/BasisType.C, line 1125:

 Setting pure/cartesian flag to 2111 for general basis


 Requested basis set is non-standard
 There are 4 shells and 4 basis functions

Total QAlloc Memory Limit   2000 MB
Mega-Array Size        61 MB
MEM_STATIC part        62 MB

                       Distance Matrix (Angstroms)
             Be(  1)
   Be(  2)  3.690000

 A cutoff of  1.0D-08 yielded      9 shell pairs
 There are         9 function pairs
 Smallest overlap matrix eigenvalue = 7.13E-01

  Scale SEOQF with 1.000000e+00/1.000000e+00/1.000000e-01

 Standard Electronic Orientation quadrupole field applied
 Nucleus-field energy     =    -0.0000000029 hartrees
 Guess MOs from core Hamiltonian diagonalization
 A restricted Hartree-Fock SCF calculation will be
 performed using Pulay DIIS extrapolation
 SCF converges when DIIS error is below 1.0E-05
 ---------------------------------------
  Cycle       Energy         DIIS Error
 ---------------------------------------
    1     -28.7014182497      9.42E-16
    2     -28.7014182497      7.38E-16 Convergence criterion met
 ---------------------------------------
 SCF time:  CPU 0.00 s  wall 0.00 s
 SCF   energy in the final basis set = -28.7014182497
 Total energy in the final basis set = -28.7014182497
 Analysis of SCF Wavefunction

 --------------------------------------------------------------
                    Orbital Energies (a.u.)
 --------------------------------------------------------------

 Alpha MOs
 -- Occupied --
 -4.483  -4.483  -0.272  -0.236
 --------------------------------------------------------------

          Ground-State Mulliken Net Atomic Charges

     Atom                 Charge (a.u.)
  ----------------------------------------
      1 Be                   -0.000000
      2 Be                   -0.000000
  ----------------------------------------
  Sum of atomic charges =    -0.000000

 -----------------------------------------------------------------
                    Cartesian Multipole Moments
 -----------------------------------------------------------------
    Charge (ESU x 10^10)
                -0.0000
    Dipole Moment (Debye)
         X       0.0000      Y       0.0000      Z       0.0000
       Tot       0.0000
    Quadrupole Moments (Debye-Ang)
        XX     -11.0827     XY       0.0000     YY     -11.0827
        XZ       0.0000     YZ       0.0000     ZZ     -11.2958
    Octapole Moments (Debye-Ang^2)
       XXX       0.0000    XXY       0.0000    XYY       0.0000
       YYY       0.0000    XXZ       0.0000    XYZ       0.0000
       YYZ       0.0000    XZZ       0.0000    YZZ       0.0000
       ZZZ       0.0000
    Hexadecapole Moments (Debye-Ang^3)
      XXXX     -18.7708   XXXY       0.0000   XXYY      -6.2569
      XYYY       0.0000   YYYY     -18.7708   XXXZ       0.0000
      XXYZ       0.0000   XYYZ       0.0000   YYYZ       0.0000
      XXZZ     -44.1207   XYZZ       0.0000   YYZZ     -44.1207
      XZZZ       0.0000   YZZZ       0.0000   ZZZZ    -246.7067
 -----------------------------------------------------------------
Archival summary:
1\1\medusa\SP\HF\BasisUnspecified\e2\yhliu\MonJun1616:40:342014MonJun1616:40:342014\0\\#,HF,BasisUnspecified,\\0,1\Be\Be,1,3.69\\HF=-28.7014182\\@

 Total job time:  0.03s(wall), 0.03s(cpu) 
 Mon Jun 16 16:40:34 2014

        *************************************************************
        *                                                           *
        *  Thank you very much for using Q-Chem.  Have a nice day.  *
        *                                                           *
        *************************************************************


